Library/Homebrew/cmd/tap-info.rb

```diff
@@ -62,13 +62,11 @@ def print_tap_info(taps)
         if tap.installed?
           info += tap.pinned? ? "pinned" : "unpinned"
           info += ", private" if tap.private?
-          if (formula_count = tap.formula_files.size).positive?
-            info += ", #{Formatter.pluralize(formula_count, "formula")}"
+          if (contents = tap.contents).empty?
```

Comment:
`info += if`?

Reply:
I think I had this before, but it failed because it calls `to_str` on `nil`.

Reply:
👍